<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>390</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QR코드 생성기</string>
  </property>
  <widget class="QLabel" name="lblQrCode">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>30</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtQrData">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTrans">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>QR코드 생성</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
